csrankings-b.csv

```diff
@@ -114,6 +114,7 @@ Barna Saha,Univ. of California - San Diego,https://barnasaha.net,BsmjRdoAAAAJ
 Barnaby Martin,Durham University,https://www.dur.ac.uk/computer.science/staff/profile/?id=15258,NOSCHOLARPAGE
 Barrett Ens,Monash University,https://research.monash.edu/en/persons/barrett-ens,HDnP1eoAAAAJ
 Barrett R. Bryant,University of North Texas,http://www.cse.unt.edu/~bryant,hMCK39sAAAAJ
+Barry Brown 0001,University of Copenhagen,https://di.ku.dk/english/staff/?pure=en/persons/742081,F59wGKwAAAAJ

```

Comment:
Also need entry for "Barry A. T. Brown".

Reply:
Same person (me).  I only have one page at DBLP though.

Reply:
You need to include entries for all DBLP aliases.

Reply:
You resolved this in error. You need to add an entry for "Barry A. T. Brown".

Reply:
Bump @barbro66